src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -670,15 +664,16 @@ impl<'a, 'b:'a, 'tcx:'b> GraphBuilder<'a, 'b, 'tcx> {
                 module_.increment_outstanding_references_for(target, ValueNS, is_public);
                 module_.increment_outstanding_references_for(target, TypeNS, is_public);
             }
-            GlobImport => {
+            GlobImport if !is_prelude => {
                 // Set the glob flag. This tells us that we don't know the
                 // module's exports ahead of time.
                 module_.inc_glob_count(is_public)
             }
+            _ => {}
```

Comment:
I would suggest avoiding `_` but rather keeping the match exhaustive, with a comment explaining why this case requires no action. This has two advantages:

- Helps the reader know why the prelude case is different.
- Gives you errors if you add new enum variants.